modules/menus/media/components/index.ts

```diff
@@ -27,7 +27,7 @@ const Media = (): BoxWidget => {
                                 hpack: 'fill',
                                 hexpand: true,
                                 vertical: true,
-                                children: [MediaInfo(), Controls(), Bar()],
+                                children: [MediaInfo(), Bar(), Controls()],
```

Comment:
Not a fan of this change. Something about the bar below the controls doesn't look appealing.